app/services/people/crud.rb

```diff
@@ -9,24 +9,20 @@ def initialize(params)
     end
 
     ATTRIBUTES = %i[first_names last_name date_of_birth gender_additional_information].freeze
-    PROFILE_ASSOCIATIONS = %i[gender ethnicity].freeze
+    PERSON_ASSOCIATIONS = %i[gender ethnicity].freeze
```

Comment:
Should this have `profiles` as well?
what about using V2::PersonSerializer::SUPPORTED_RELATIONSHIPS ?

Reply:
If I add profiles this would change the functionality of the controllers. Leaving for moment and we might update this as part of adding support to create/update a person with a profile in future.